### PR TITLE
Sort Filter Api Contract 

### PR DESCRIPTION
**Pull request by tavisca-acharpe**

### CODE REVIEW

File: Contracts/v1.0/buyer/ad-list/request.json5 [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
 
 //Header contains "sessionId": "FAE04EC0-301F-11D3-BF4B-00C04F79EFBC" and "userId": "1111"
+//Request object
+{
+    //[Optional] sort by 
+    "sortBy" : "date",
+    //[Optional] order of Sorting
+    "sortOrder" : "ascending",
+    //[Optional] query for search filter etc
+    "filter" :[
```
Comment (Tavisca-smohite) [inline Contracts/v1.0/buyer/ad-list/request.json5]: `filters`

File: Contracts/v1.0/buyer/ad-list/request.json5 [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+	   {
+		    "searchKeyword":"watch"
```
Comment (Tavisca-smohite) [inline Contracts/v1.0/buyer/ad-list/request.json5]: use something else

File: Contracts/v1.0/buyer/ad-list/request.json5 (hunk 1 above)
Comment (tavisca-acharpe) [reply Contracts/v1.0/buyer/ad-list/request.json5]: done

File: Contracts/v1.0/buyer/ad-list/request.json5 (hunk 2 above)
Comment (tavisca-acharpe) [reply Contracts/v1.0/buyer/ad-list/request.json5]: 1.searchKeyword
2.searchQuery
3.searchIdentifier etc

plz suggest something

**Review by achirametavisca — APPROVED** [review]
Merge

File: Contracts/v1.0/buyer/ad-list/request.json5 [hunk 3]
```diff
@@ -1,3 +1,23 @@
 //GET /api/v1.0/{applicationname}/products?pagenumber=1&pagesize=50
 
 //Header contains "sessionId": "FAE04EC0-301F-11D3-BF4B-00C04F79EFBC" and "userId": "1111"
+//Request object
+{
+    //[Optional] sort by 
+    "sortBy" : "date",
+    //[Optional] order of Sorting 
+    "sortOrder" : "ascending",
```
Comment (Tavisca-smohite) [inline Contracts/v1.0/buyer/ad-list/request.json5]: make it `orderBy` then

**Review by Tavisca-smohite — APPROVED** [review]

**Pull request merged**